### PR TITLE
Add AVIF_ENABLE_EXPERIMENTAL_YCGCO_R=ON to CI

### PR DESCRIPTION
**Pull request by y-guyon**

### CODE REVIEW

File: src/reformat.c [hunk 1]
```diff
@@ -749,9 +749,9 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image,
                     const int Cg = (int)avifRoundf(Cb * yuvMaxChannel);
                     const int Co = (int)avifRoundf(Cr * yuvMaxChannel);
                     const int t = YY - (Cg >> 1);
-                    G = AVIF_CLAMP(t + Cg, 0, state->rgbMaxChannel);
-                    B = AVIF_CLAMP(t - (Co >> 1), 0, state->rgbMaxChannel);
-                    R = AVIF_CLAMP(B + Co, 0, state->rgbMaxChannel);
+                    G = (float)AVIF_CLAMP(t + Cg, 0, state->rgbMaxChannel);
+                    B = (float)AVIF_CLAMP(t - (Co >> 1), 0, state->rgbMaxChannel);
+                    R = (float)AVIF_CLAMP(B + Co, 0, state->rgbMaxChannel);
```
Comment (wantehchang) [inline src/reformat.c:754]: These changes should be moved to a separate CL.

**Review by wantehchang — APPROVED** [review]
LGTM. The pull request contains seemingly unrelated changes.

It would be good to explain in the commit message how you chose the CI builds to have `AVIF_ENABLE_EXPERIMENTAL_YCGCO_R=ON`.

**Comment by y-guyon** [comment]
> LGTM. The pull request contains seemingly unrelated changes.

The changes were to make the CI pass on Windows.

> It would be good to explain in the commit message how you chose the CI builds to have `AVIF_ENABLE_EXPERIMENTAL_YCGCO_R=ON`.

It is good to test with and without `AVIF_ENABLE_EXPERIMENTAL_YCGCO_R` so I enabled it in a few diverse platforms.
I can add that to the commit message when rebasing after https://github.com/AOMediaCodec/libavif/pull/1495.
